packages/glibc.rb

```diff
@@ -3,25 +3,30 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version '1.1'
+  version '1.2'
 
   is_fake
 
-  if (File.exist? "/lib/libc-2.23.so") || (File.exist? "/lib64/libc-2.23.so")
-    depends_on 'glibc223'
-    # Check previous version
     if File.exist? CREW_CONFIG_PATH + "meta/glibc219.filelist"
       conflict_solve = "`crew remove glibc219`"
+  else
+      depends_on 'glibc223'
```

Comment:
I don't think you need this.  The next conditional will cover the `depends_on 'glibc223'`.